common/src/test_utils.rs

```diff
@@ -33,8 +33,18 @@ pub fn generate_test_state_from_workloads(workloads: Vec<WorkloadSpec>) -> State
         api_version: API_VERSION.into(),
         workloads: workloads
             .into_iter()
-            .map(|v| (v.instance_name.workload_name().to_owned(), v.into()))
+            .map(|v| {
+                let name = v.instance_name.workload_name().to_owned();
+                let mut w = StoredWorkloadSpec::from(v);
+                w.configs = [
+                    ("ref1".into(), "config.path.1".into()),
+                    ("ref2".into(), "config.path.2".into()),
+                ]
+                .into();
+                (name, w)
+            })
             .collect(),
+        configs: HashMap::new(),
```

Comment:
Why is the workload referencing some config items but in the `configs` field they do not exist? My expectation to the `generate_test_...` functions is that I get always valid test dummy data.

Below are also other functions that contain references to configs but no configuration.

Reply:
fixed

Reply:
Changes are fine.